gql/baseResolver.js

```diff
@@ -23,28 +25,28 @@ class BaseResolver {
     this.schemas = { User }
 
     const baseErrors = {
```

Comment:
would adding status codes to this be beneficial ?

Reply:
Nope, discussed